kotlin-react/src/main/kotlin/react/useEffect.kt

```diff
@@ -1,6 +1,8 @@
 package react
 
-typealias RDependenciesArray = Array<out dynamic>
+import kotlinext.js.ReadonlyArray
+
+typealias RDependenciesArray = ReadonlyArray<dynamic>
```

Comment:
this type is used not only in effect hooks, maybe it is good opportunity to move it in some common place?

Reply:
It has sense to inline it in separate PR